src/components/EventDashboard/EventDashboardPage.tsx

```diff
@@ -0,0 +1,138 @@
+  Container,
+  Tabs,
+  Tab,
+  makeStyles,
+} from '@material-ui/core';
+import { useQuery } from '@apollo/react-hooks';
+import { RouteComponentProps } from 'react-router';
+import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
+import LocationOnOutlinedIcon from '@material-ui/icons/LocationOnOutlined';
+import CCPTabPanel from './CCPTabPanel';
+import MenuAppBar from '../common/MenuAppBar';
+import { Colours } from '../../styles/Constants';
+import { GET_EVENT_BY_ID } from '../../graphql/queries/events';
+import ResourceTabPanel from './ResourceTabPanel';
+import { formatDate } from '../../utils/format';
+import LoadingState from '../common/LoadingState';
+
+type TParams = { eventId: string };
+
+export enum TabOptions {
+  CCP = 0,
+  Hospital = 1,
+  Ambulance = 2,
+}
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: any;
+  value: TabOptions;
+}
+
+const useStyles = makeStyles({
+  root: {
+    minHeight: '100vh',
+    background: Colours.BackgroundGray,
+  },
+  container: {
+    background: Colours.White,
+    padding: '32px 56px',
+    maxWidth: 'none',
+  },
+  tabs: {
+    background: Colours.White,
+    padding: '0 56px',
+  },
+  icon: { fontSize: '18px', paddingRight: '10px' },
+});
+
+const TabPanel = (props: TabPanelProps) => {
+  const { children, value, index, ...other } = props;
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      {...other}
+    >
+      {children}
+    </div>
+  );
+};
+
+const EventDashboardPage = ({ match }: RouteComponentProps<TParams>) => {
+  const classes = useStyles();
+  const { eventId } = match.params;
+  // TO DO: error handling when eventId does not exist in database
+  const { loading: eventLoading, data: eventInfo } = useQuery(GET_EVENT_BY_ID, {
+    variables: { eventId },
+  });
+
+  const [tab, setTab] = React.useState(TabOptions.CCP);
+
+  if (eventLoading) {
+    return <LoadingState />;
+  }
+
+  const { event } = eventInfo;
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: TabOptions) => {
+    setTab(newValue);
+  };
+
+  return (
+    <Box className={classes.root}>
+      <MenuAppBar pageTitle="Directory" eventId={eventId} />
+      <Container className={classes.container}>
+        <Typography variant="h3">{event.name}</Typography>
+        <Box
+          display="flex"
+          alignItems="center"
+          style={{
+            color: Colours.SecondaryGray,
+          }}
+        >
+          <LocationOnOutlinedIcon className={classes.icon} />
+          <Typography variant="body1" style={{ paddingRight: '52px' }}>
+            Ezra Street l1j3j4, Waterloo Canada
+          </Typography>
+          <CalendarTodayOutlinedIcon className={classes.icon} />
+          <Typography variant="body1">{formatDate(event.eventDate)}</Typography>
+        </Box>
+      </Container>
+      <Tabs
+        textColor="secondary"
+        className={classes.tabs}
```

Comment:
should indicator be thicker to be more consistent with other tabs designs?

Reply:
I think we'll leave this for now, because we will likely refactor all the tabs in our app to use `MenuTabs` in the `common` components folder.